alacritty/src/event.rs

```diff
@@ -1546,6 +1505,30 @@ impl Processor {
 
                     info!("Initialisation complete");
                 },
+                // NOTE: Request a redraw for terminal wakeup to minimize the input latency.
```

Comment:
This is written like a normal comment, but has a `NOTE` label. The two don't really fit together.

Either reword the comment or remove the label.